gradlew

```diff
@@ -1,172 +0,0 @@
-#!/usr/bin/env sh
-
-##############################################################################
-##
-##  Gradle start up script for UN*X
-##
```

Comment:
pretty sure this is autogenerated by the gradle tool. I think when you run the gradle wrapper command which android studio does behind the scenes somewhere this script is pasted into the directory.

Reply:
Auto generated by android studio.

Reply:
👍